tf2onnx/onnx_opset/tensor.py

```diff
@@ -835,7 +859,11 @@ def version_10(cls, ctx, node, **kwargs):
                     unqueeze_at.append(bit)
                     begin_mask |= 1 << bit
                     end_mask |= 1 << bit
-            input_x = ctx.make_node("Unsqueeze", [input_x.output[0]], {"axes": unqueeze_at})
+            if opset < 13:
```

Comment:
Again unqueeze_at is const so you should just be able to replace the if/else with:
`GraphBuilder(ctx).make_unsqueeze('data': input_x.output[0], 'axes': unqueeze_at)`